.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: cpp
+dist: trusty
+os: linux
+cache: apt
+branches:
+  only:
+    - master
+matrix:
+  include:
+   - env: BUILD_TYPE=RelWithDebInfo
+     script:
+      - docker build -t verify-c-common -f verify-c-common.Dockerfile .
+      - docker run --rm -it verify-c-common
```

Comment:
something like this will be more clear
`docker run -t verify-c-common /bin/bash -c  "cd build && ninja test"`